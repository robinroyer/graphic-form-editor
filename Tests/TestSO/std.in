R r01 10 10 5 5
R r02 20 20 10 10
R r03 10 10 5 5
R r04 20 20 10 10
R r05 10 10 5 5
R r06 20 20 10 10
S s01 0 0 30 30
LIST
DELETE r01
LIST
DELETE r02 r03
LIST
DELETE s01
LIST
EXIT