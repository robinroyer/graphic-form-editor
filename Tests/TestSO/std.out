OK
#New object: r01
OK
#New object: r02
OK
#New object: r03
OK
#New object: r04
OK
#New object: r05
OK
#New object: r06
OK
#New object: s01
R r01 10 10 5 5
R r02 20 20 10 10
R r03 10 10 5 5
R r04 20 20 10 10
R r05 10 10 5 5
R r06 20 20 10 10
OK
#Objects sucsessfully removed
R r02 20 20 10 10
R r03 10 10 5 5
R r04 20 20 10 10
R r05 10 10 5 5
R r06 20 20 10 10
OK
#Objects sucsessfully removed
R r04 20 20 10 10
R r05 10 10 5 5
R r06 20 20 10 10
OK
#Objects sucsessfully removed
